docs/installation/server-config/security.rst

```diff
@@ -3,8 +3,11 @@
 Configuring Security for KSQL
 =============================
 
-KSQL supports many of the security features of both Apache Kafka and the |sr|.
+KSQL supports authentication on its http endpoints and also supports many of the security features
+of the other services it communicates withof both Apache Kafka and the |sr|.
 
+- KSQL supports BASIC http authentication on its RESTful and Websocket HTTP endpoints, i.e. the
```

Comment:
Replace "i.e." with something like "... endpoints, which means that the..." (Better for localization, if we ever decide to do that.)